src/cli.ts

```diff
@@ -145,22 +74,29 @@ export function startServer(relativePath: string, bundle: string, meta: string,
     fs.removeSync(path.join(process.cwd(), './src/test-temp'))
     fs.mkdirSync(path.join(process.cwd(), './src/test-temp'))
     fs.mkdirSync(path.join(process.cwd(), './src/test-temp/languages'))
```

Comment:
lets use the `build/languages` directory here. In order for the languageLanguage to use this path you must pass `{"storagePath": "/complete/path/build/languages"}` into the `languageLanguageSettings` section in the bootstrap seed file.